### PR TITLE
Not found middleware

### PR DESCRIPTION
**Pull request by JoeChapman**
- add not-found with tests
- export as notFound
- uipdate readme to document

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,5 +2,6 @@
 
 module.exports = {
   cookies: require('./lib/cookies'),
-  errors: require('./lib/errors')
+  errors: require('./lib/errors'),
+  'notFound': require('./lib/not-found')
```
Comment (joefitter) [inline index.js]: superfluous `'`'s

File: lib/not-found.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const getTranslations = (translate) => {
```
Comment (joefitter) [inline lib/not-found.js]: we don't need these `(` `)`'s. Not saying we should definitely remove them, guess its a preference thing but we should be consistent.

File: lib/not-found.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  if (translate) {
+    translations.title = translate('errors.404.title');
+    translations.description = translate('errors.404.description');
+  }
+
+  return translations;
+};
+
+module.exports = (options) => {
```
Comment (joefitter) [inline lib/not-found.js]: and here

**Comment by JoeChapman** [comment]
@joefitter @easternbloc

**Comment by joefitter** [comment]
:shipit:

**Comment by JoeChapman** [comment]
@easternbloc @daniel-ac-martin @bendiggle @gxxm

**Comment by JoeChapman** [comment]
In case you have forgotten: @joefitter @gxxm @bendiggle @easternbloc @daniel-ac-martin this needs some eye love!

**Comment by gxxm** [comment]
:thumbsup:

**Pull request merged**